src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -683,6 +708,7 @@
   },
   {
     "id": "TA-000026",
+    "name": "ServiceFabric.OnlyClientAADAuth",
```

Comment:
I think "OnlyAADClientAuth" parses better in my head for saying that "you can only use AAD for client authentication". 

Also, after looking deeper into the rule when I was thinking about the name, the rule itself seems like it is only checking that AAD client auth is enabled (by checking properties.azureActiveDirectory.tenantId has a value). If it was the "only" auth shouldn't [properties.clientCertificateThumbprints and properties.clientCertificateCommonNames](https://learn.microsoft.com/en-us/azure/templates/microsoft.servicefabric/clusters?pivots=deployment-language-arm-template#property-values-1) be considered in the rule if we're saying "only" AAD client auth (and not client cert auth)? Might need to discuss a rule update/fix if so.

Reply:
:O do you want to add an issue?

Reply:
#299